src/frontend/js/index.js

```diff
@@ -163,7 +165,7 @@ function zones2summary(summariser, state) {
     return R.pipe(summariser,x=>numberToHuman(x, state))(
         state.selectedZones.length > 0 ?
         R.pipe(R.pickAll,R.values)(
-            R.map(R.add(-1),state.selectedZones), state.layers.od_matrices.basevalues
```

Comment:
I'm not sure whether we should be subtracting one any more. I'm assuming that the new Julia backend will supply a dict with `(fid => sum)` rather than the current `array[fid + 1] = sum`.

As it stands it seems like removing the -1 from other parts of the code leads to weird data.